client/app/components/user/session/user-session.service.js

```diff
@@ -406,141 +414,148 @@ class SessionService {
   }
 
   getUserMenu(currentUser) {
-    return {
-      name: 'user',
-      title: currentUser.firstName,
-      iconClass: 'icon-user',
-      nichandle: currentUser.nichandle,
-      fullName: `${currentUser.firstName} ${currentUser.lastName}`,
-      subLinks: [
+    const useNewText = ['FR'].includes(currentUser.ovhSubsidiary);
+
+    const textTitle = useNewText
```

Comment:
Bit weird to have the condition done twice, you could:

```suggestion
    const p = useNewText ? this.Navbar.buildMenuHeader(`${this.$translate.instant('otrs_menu_userAccount_1', { username: currentUser.firstName })}<br>${this.$translate.instant('otrs_menu_userAccount_2')}`) : this.$q.when(currentUser.firstName)
```